DeploymentCloud/Deployment.DataX/deploySample.ps1

```diff
@@ -179,12 +199,15 @@ function Add-CosmosDBData ([string]$filePath) {
 function Get-Tokens {
     $tokens = Get-DefaultTokens
 
-    $tokens.Add('subscriptionId', $subscriptionId )
+    $tokens.Add('subscriptionId', $subscriptionId)
     $tokens.Add('resourceLocation', $resourceGroupLocation)
     $tokens.Add('iotHubName', $iotHubName)
     $tokens.Add('kafkaName', $kafkaName)
     $tokens.Add('kafkaEventHubNamespaceName', $kafkaEventHubNamespaceName)
     $tokens.Add('clientSecretPrefix', $clientSecretPrefix)
+
+    $deploymentDate = Get-Date -Format "yyyy-MM-dd"
```

Comment:
When running the ARM template if the date changes, will the sample still work? eg. ARM template is started at 11 PM and it completes at 2 AM the next day, will the sample still work ?

Reply:
yes it will still work. This datetime will be cached only when the deploysample.ps1 executes.